assets/src/edit-story/components/panels/index.js

```diff
@@ -120,10 +136,8 @@ export function getPanels(elements, options = {}) {
     // If the selected element's type is video / image , display accessibility panel, too.
     if ('video' === elements[0].type) {
       panels.push({ type: VIDEO_OPTIONS, Panel: VideoOptionsPanel });
-      panels.push({
-        type: CAPTIONS,
-        Panel: CaptionsPanel,
-      });
+      panels.push({ type: POSTER, Panel: PosterPanel });
```

Comment:
Let's call this `VideoPosterPanel` to disambiguate with "story poster".

Reply:
Got it, updated.